src/game/scenes/mainmenu.c

```diff
@@ -138,6 +138,7 @@ int mainmenu_create(scene *scene) {
     scene_set_dynamic_tick_cb(scene, mainmenu_tick);
     scene_set_startup_cb(scene, mainmenu_startup);
 
+#ifdef DEBUGMODE
     if(scene->gs->net_mode == NET_MODE_CLIENT) {
         component_action(guiframe_find(local->frame, NETWORK_BUTTON_ID), ACT_PUNCH);
         component_action(guiframe_find(local->frame, NETWORK_CONNECT_BUTTON_ID), ACT_PUNCH);
```

Comment:
Please do not break the net_mode field, this is set via commandline options that this PR is not removing.
I think leaving the network game launch options in is fine, as most users will not discover them-- and they will probably be useful when we need to check if netplay works with CI pipeline-built binaries.

Reply:
Okay, I let the `scene->gs->net_mode = NET_MODE_NONE;` stay, but it doesn't affect anything, as it is 0 anyway. About the CI build, that uses debug mode for testing, so don't need it there..

Reply:
Oh yeah, and disabled the command line stuff too here.